Source/WebKit/UIProcess/Launcher/cocoa/ProcessLauncherCocoa.mm

```diff
@@ -56,8 +56,66 @@
 #import "CodeSigning.h"
 #endif
 
+#if USE(EXTENSIONKIT)
+#import "ExtensionKitSPI.h"
+
+SOFT_LINK_FRAMEWORK_OPTIONAL(ServiceExtensions);
+SOFT_LINK_CLASS_OPTIONAL(ServiceExtensions, _SEServiceConfiguration);
+SOFT_LINK_CLASS_OPTIONAL(ServiceExtensions, _SEServiceManager);
+#endif
+
 namespace WebKit {
 
+#if USE(EXTENSIONKIT)
+static std::pair<const char*, RetainPtr<NSString>> serviceNameAndIdentifier(ProcessLauncher::ProcessType processType)
```

Comment:
Should this maybe just be an Objective C NSDictionary, since it's only really needed for Cocoa APIs?

Reply:
Actually, maybe not since the key is used for regular WebKit C++ code.

Reply:
Please use ASCIILiteral instead of `const char*`, then thread-safety is not a concern.

Reply:
That is a good point! I will change this to `ASCIILiteral`.

Thanks for reviewing!